crates/ironrdp-rdpsnd/src/lib.rs

```diff
@@ -0,0 +1,54 @@
+use ironrdp_pdu::{gcc::ChannelName, write_buf::WriteBuf, PduResult};
+use ironrdp_svc::{AsAny, CompressionCondition, StaticVirtualChannel};
+use std::any::Any;
+use tracing::warn;
+
+/// We currently don't implement any of rdpsnd, however it's required
+/// for rdpdr to work: https://tinyurl.com/2fvrtfjd
```

Comment:
Even if the original link is very loong, it’s probably better to still not shorten it:
- tinyurl.com is sometimes blocked for security reasons
- They say it will never expire, but you never know